MetaMask/metamask-extension
# UX: Icon: Remove Plus icon

## Explanation

Removes the legacy plus icon and component

## Screenshots/Screencaps

<img width="426" alt="NewPlusIcon" src="https://user-images.githubusercontent.com/46655/217664164-eb93b8a9-1a0e-4c77-8ced-109d2fed8009.png">


## Manual Testing Steps

1. Open the account menu
2. See new + icon

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
